src/Laravel/Casts/Int8Cast.php

```
@@ -0,0 +1,55 @@
+    }
+
+    /**
+     * Prepare the given value for storage.
+     *
+     * @param Model $model
+     * @param string $key
+     * @param Int8|mixed $value
+     * @param array $attributes
+     * @return int|null
+     */
+    public function set($model, $key, $value, $attributes): ?int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (undefined)

(UnusedFormalParameter)